scripts/publish

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Nit: `#! /usr/bin/env bash` is most portable https://stackoverflow.com/q/16365130